src/components/Matching/index.tsx

```diff
@@ -0,0 +1,68 @@
+import React, { useState } from 'react'
+import { Route, Switch } from 'react-router-dom'
+import { StyledLink } from '../StyledText/StyledLink'
+import { StyledButton, Wrapper, TitleTabs, ContentTabs } from './styled'
+
+type TabsType = {
+	title: string
+	route: string
+}
+
+const dataTabs: TabsType[] = [
+	{
+		title: 'Mentor',
+		route: '/matching/mentor',
+	},
+	{
+		title: 'Buddy',
+		route: '/matching/buddy',
+	},
+	{
+		title: 'Mentee',
+		route: '/matching/mentee',
+	},
+]
+
+const MatchingTabs: React.FC = () => {
+	const [active, setActive] = useState(dataTabs[1])
+
+	return (
+		<Wrapper>
+			<TitleTabs>
+				{dataTabs.map((tab, index) => (
+					<StyledLink key={index} to={tab.route}>
+						<StyledButton
+							title={tab.title}
+							active={active === tab}
+							onClick={() => setActive(tab)}
+						/>
+					</StyledLink>
+				))}
+			</TitleTabs>
+			<ContentTabs>
```

Comment:
What are we planning in the future? Each tab will have it's own component or is one component in something like a map enough? :thinking: @macajind

Reply:
I was thinking about the function of maps, but it seemed complicated or at least confusing for me. Now we know what to put. But if the map is better, I'll just remake it 👍

Reply:
I used very similar map in Skillqueue (see PR #121), I think, for displaying three rows, each with different type of skills. Maybe it could work for this as well.

Reply:
I think this is fine for the first faze. We'll make it dynamic when it's gonna be necessary. :wink: